docs/to_clarify.md

```diff
@@ -1 +1,23 @@
-1. Why does identification verification require "booking document + passport + credit card authorization"?
+1. Stated under the section "Problem Statement" that the solution must be "safe and secured to protect guests' privacy". What exactly does this requirement mean, for example, are there certain security standards that the solution must adhere to?
+
+1. Stated under the section "Possible Use Cases" that the solution should "enhance back-end operational capabilities and business intelligence" which will be used to "create accurate guest profiles and provide personalized and targeted guest experiences". In what manner is the solution expected to do this?
+
+1. Stated under the section "Possible Use Cases" that the solution should "allow data transference capabilities, from room requests collected on the booking engine, to the check-in process till checkout".
+
+    Does this mean that the "digital mobile check-in and check-out solution" is expected to do more than digitising the checking-in and checking-out processes; it has to integrate with other platforms to provide other additional services, beginning from the booking of hotel rooms till checkout?
```

Comment:
I think we have many questions concerning the integration with other systems. Should we gather all of them under one section for questions on integration?

Reply:
Yup, we should. I'll create an issue so that we can collate our questions on integration and we remove the segments in our PRs, then someone can create a separate PR just for integration.